code/game/objects/items/storage/boxes.dm

```diff
@@ -966,3 +966,47 @@ obj/item/storage/box/clown
 /obj/item/storage/box/holy_grenades/PopulateContents()
 	for(var/i in 1 to 7)
 		new/obj/item/grenade/chem_grenade/holy(src)
+
+/obj/item/storage/box/stockparts //for ruins where it's a bad idea to give access to an autolathe/protolathe, but still want to make stock parts accessible
+	name = "box of stock parts"
+	desc = "Contains a variety of basic stock parts."
+
+/obj/item/storage/box/stockparts/basic

```

Comment:
Any reason this isn't just part of `box/stockparts`? Not sure it needs to change, just wondering if you had a reason.

Reply:
I did the parent/subtype thing out of habit, no real reason behind it. Want me to change it?

Reply:
Might as well. I get the feeling that having fewer redundant typepaths floating about the codebase is a good idea.